core/dbt/contracts/graph/manifest.py

```diff
@@ -1007,10 +1006,7 @@ def add_exposure(self, source_file: SchemaSourceFile, exposure: ParsedExposure):
         source_file.exposures.append(exposure.unique_id)
 
     def add_disabled_nofile(self, node: CompileResultNode):
-        if node.unique_id in self._disabled:
-            self._disabled[node.unique_id].append(node)
-        else:
-            self._disabled[node.unique_id] = [node]
+        self.disabled.append(node)
```

Comment:
What's changed here? Are we no longer patching properties for disabled nodes?

Reply:
I don't think we've ever patched disabled nodes. That's the way that it worked when I first saw it anyway.

Reply:
I'm not sure that feels right though. I woudn't have no objection to patching disabled nodes. They do come out in the manifest.json.

Reply:
But a lookup won't find them (since they're not included there), so we'd have to separately look them up in disabled.